test/Engine/ProtoTest/Imperative/MicroFeatureTests.cs

```diff
@@ -451,7 +451,7 @@ public void ForLoop05()
             thisTest.RunScriptSource(code);
             thisTest.Verify("y",11);
         }
-        [Ignore]
+        [Ignore("unknown reason")]
```

Comment:
the empty constructor for this attribute was removed, we now have to use the overload that accepts a string param, but now it will work with either nunit2 or 3.